tests/cryosms.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from utils.test_modes import TestModes
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.testing import get_running_lewis_and_ioc
+
+DEVICE_PREFIX = "CRYOSMS_01"
+EMULATOR_NAME = "HFMAGPSU"
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("CRYOSMS"),
+        "emulator" : EMULATOR_NAME,
+    },
+]
+
+
+TEST_MODES = [TestModes.DEVSIM]
+
+
+class CryoSMSTests(unittest.TestCase):
+    def setUp(self):
+        self._lewis, self._ioc = get_running_lewis_and_ioc(EMULATOR_NAME, DEVICE_PREFIX)
+        self.ca = ChannelAccess(device_prefix=DEVICE_PREFIX, default_timeout=10)
+
+        self.ca.assert_that_pv_exists("DISABLE", timeout=30)
+
+    def test_WHEN_ioc_is_started_THEN_it_is_not_disabled(self):
+        self.ca.assert_that_pv_is("DISABLE", "COMMS ENABLED")
```

Comment:
Why do we need this test?

Reply:
This was from the example file. Removed it here, though will be making a similar check in the next CRYOSMS ticket.